Tasks/Common/webdeployment-common/xmlvariablesubstitutionutility.ts

```diff
@@ -262,4 +265,18 @@ function updateXmlConnectionStringsNodeAttribute(xmlDomNode, variableMap, replac
     }
 
     return isSubstitutionApplied;
+}
+
+function replaceEscapeXMLCharacters(xmlDOM) {
```

Comment:
Should we rename xmlDOM to xmlDomNode in method argument?

Reply:
renamed.